pythonforandroid/patching.py

```diff
@@ -69,3 +70,17 @@ def is_ndk(ndk):
     def is_x(recipe, **kwargs):
         return recipe.ctx.ndk == ndk
     return is_x
+
+def is_version_gt(version):
+    def is_x(recipe, **kwargs):
+        return LooseVersion(recipe.version) > version
+
+def is_version_lt(version):
+    def is_x(recipe, **kwargs):
+        return LooseVersion(recipe.version) < version
+    return is_x
```

Comment:
I think that we could remove this two methods, since we don't use it anywhere...or maybe you were planning to use that in a near future...in that case just fixing the [the gh-actions pep8 error](https://github.com/kivy/python-for-android/pull/2044/checks?check_run_id=444633138#step:4:16) should be enough to make happy travis and gh-actions.

**Note:** if you remove the methods, you should also probably remove the import line: `from distutils.version import LooseVersion`

Reply:
I'd prefer to leave these in, since the patching.py is a set of generic comparison tools that we might want to use in the future, but I've added some blank lines to make pylint happy.